packages/portalnetwork/src/subprotocols/history/headerAccumulator.ts

```diff
@@ -77,6 +82,27 @@ export class HeaderAccumulator {
     }
     return this._currentEpoch.push(headerRecord) - 1
   }
+  /**
+   * Adds a new block header to the `currentEpoch` in the header accumulator
+   */
+  public updateAccumulatorFromEthers = (newHeader: { hash: string; difficulty: number }) => {
```

Comment:
Do we need this anymore?  Seems like it's of limited value since we have a hardcoded master accumulator.  Can't we rip out all of this logic for updating the master accumulator?

Reply:
we can.  What seemed like minor changes had already created a monster PR.  But yes, ripping all of that out is fine now that we have the pre-merge accumulator set in stone

Reply:
Went ahead and axed all of the updating and snapshotting stuff.